packages/react-button/src/components/SplitButton/SplitButton.tsx

```diff
@@ -11,7 +11,7 @@ import { MenuButton } from '../MenuButton/index';
 export const SplitButton = React.forwardRef<HTMLElement, SplitButtonProps>((props, ref) => {
   const { state, render } = useSplitButton(props, ref, {
     button: { as: Button },
-    menuButton: { as: MenuButton, iconOnly: true, icon: <ChevronDownIcon /> },
+    menuButton: { as: MenuButton, menuIcon: <ChevronDownIcon /> },
```

Comment:
hmm. Was this intentional?

Reply:
yes. in fact it should be even simpler: `menuButton: { as: MenuButton }`. updated

Reply:
reverted